### ISSUE TITLE
10 剑4-10写作

### ISSUE THREAD

**Issue by peimin** [peimin/ielts#10]
##### 剑 10 test 1

Nowadays more and more people think that it is necessary for children to tell right and wrong. some poeple argue that punishment is a good method of teaching them this distinction. But others donot think so. My firm conviction is that children should learn the difference between right and wrong, but not by punishing them.

On the one hand, punishment might have a negative influence on children`s physical and psychological development. Psysically, punishment might cause any hurt on their body. Mentally, punishment from their teachers and parents might make they suffer from metal and emotional pressure and stress that might make they so scary and unconfident when making friends or getting along with others.

On the another hand, punishment might make our society unhealter. The child will be our future. If punishment affect badly children`s growth and development, they might donot behave well so that there might be more criminals and violence in the society. Finally, if that happened, we would live in a unhealthy and undevelopment society.

From my point of view, there are lots of efficient ways to teach children about the difference between right and wrong rather than pushing them. Tell some stories about right and wrong, rewarding them for good heaviour and correcting when they make mistakes are good ways to achieve this. Of course, it might be helpful to teach them piack up rubbish they have dropped, clean up graffiti they haved drawn or apologise to someone they have hurt. Those efficient methods will lead better future behavior than does punishment.

Finally, to help the children learn the difference between right and wrong, teachers and parents have a responsibility to provide good role modeling in their own behaviour. Only by taking good ways can they teach children efficiently.

**Comment by peimin** [comment]
When it comes to what subjects that university students should study. Some think that all university students shoud learn whatever they like. Others believe that students should subjects that are useful in the future. I agree with view that we , as university students, should study useful subjects in the future， but not neglect some subjects that will have a positive imfluence in the long-term.

First of all, consider that subjects that they are interested in. When students study in subjects that they are really interested in, they will make their every effort to do it as a result they are easier to success in those subjects. After all, interest is the best teacher. 

Secondly, consider that subjects like those related to sicence and tethnology that will be useful in the future.  
It is those subjects that makes us find jobs, raise family and lead a good life in the future. Such, those subjects have a critical role in our daily life and success. 

From my point of view, we should work hard to learn not only those subjects we are intertested in, but also job-realted subjects that are important for our daily life in the future. We should learn how to plan our times to learn those subjects and have a balance of learning those subjects. 

In a nutshell, we should learn how to plan our time of studying those subjects and try our best to learn those subjects in university. Only by doing so we can behave well  and achieve our goals in university. Most things in moderation are healthy, learning subject in university are no different.

**Comment by peimin** [comment]
we live in an era of globalisztion and countries are becoming more and more similar. we can buy the same products wherever we are. When weighting the pros and cons, it is clear that there are negatives, but the benefits clearly outweight them.

First of all, consider that benefits of globalization. On the on hand, as a result of globalisztion, we can buy products we want to wherever we are without going to the places where products are produced, which makes our life more convenient. On the other hand, we will feel similar when we travel abroad and see so many alike goods.

The flipside though is that globalisztion has a negative impact on local culture because they often substitute local traditonal products. When people travel abroad they usually want to do as the Romans do and go to the local souvenir shops to buy some local products; how unfortunate if there is no local productes left to sll. In my living city, Chengdu, local handicraft have become harder to find as lots of products from domestic and abroad have gradually taken over.

From my point of view, globalization brings some many convinience to us. It makes us can buy lots of things produced abroad without travel abroad, which makes our life more colorful. Meanwhile, we should not neglect our traditional culture and hanicraft that play a important role in our life. The more traditional culture and handicraft we have, the more colorful our life will be.

In a nutshell, with the globalization we can buy products from all over the world, but we should not neglect of the importance of traditional products. Only by combining both abroad products and local products can we make our colorful.

**Comment by peimin** [comment]
##### 剑 10 test 4
Nowadays more and more visit the musuem to see nature, science and history. Many museums arenot free for admission. However, whether all meseums should be free for admission is a controversial issue. I agree with the view that meseums should be free for admission.

First of all, let`s consider that purpose of museums is to provide knowledeg of nature, sicence and history for public. The more open museums are, the more people will go to visit. So, the managers of museums should make their every effort to ensure everyone visit museums freely.

Secondly, Mesumes is a part of public services. Governments have a responsibility to provide public services, mesumes are not different. So, governments should pay for the cost of museums and provide better public services for citizens.

On the flipside, some people think museums should charge for admission. Let`s consider that if museums arenot free, actually the expand of government on museums will reduce, but the number of people who visit museums might reduce, which is opposed to the purpose of building museums. Such, free of museums for admission cons outweight pros.

In a nutshell, museums should be free for public and governments should shouder their responsibility to pay for the cose of museums.

**Comment by peimin** [comment]
##### 剑9 test 1
With the globalisation, more and more people learn and speak English as a secondary language. However, whether children should begin learning foreign language at primary school rather than secondary school is a controversial issue. My firm conviction is that children should begin learning foreign language at primary school.

One the on hand, language acquisition is considerably easier in your first several years of life than later in life. This is a concept known in hunman biology and languistics as the critical period. So, the earlier one begins to learn a language the easier the task will be and the greater the degree of fluency you can likely attain.

One the other hand, children at primary school have more time to learn foregin language than at secondary school. For learning language, the more time you spend on speaking, writing, reading and listening this, the more fluent you will be. When children are at primary school, there are less subjects to learn, but more free time to do what they want to do. So, they can spend lots of time on improving their ability of foreign language, like listening to and telling each other short funny stories, connecting to friends who speak this language as native.

One the flip side, this is a problem that how much they can spend on learning foreign language if they begin learning it at secondary school. There are lots of scientific subjects, like physical, chemistry and so on, that they must make their effort to learn so that they have less time to learn foreign language. 

In a nutshell, the earlier the better for learning language. Young children`s brains are receptive to language development. Of courese, parents and teachers have a responsibility to lead them a better way of how to learn efficiently foreign language.

**Comment by peimin** [comment]
##### 剑9 test 2
##### 高中生是否应该设置一些无报酬的
Nowadays more and more students take part in some unpaid community services and voluntary activities. However, wheter those unpaid community services should be a compulsory part of their courses in high school is a conroversial issue. My firm conviction is that it is necessary for students to take some courses that are some unpaid community services.

One the one hand, community works can help them become a responsible person in society in the future. By taking part in unpaid job, students not only have a helping attitude, but also have a opportunity to comprehend the real situation of community. For example, if a child is taught to be helpful and explained the importance of moral values in society, he/she will have positive thoughts of our neighbor and wish to promote the community.

On the other hand, community works can assist them for their physical and psychological development. When participating in these activities, they are motivated to partcipating in various decison that will make them build strong relationship, develop team work skills and learn how to experience and interact what they want to do with their partners as a result they can success in the days to come. 

Furthermore, that is not to say that students should partciple in all kinds of community services.  As a students, there are lots of theoretical majors such as math and physics that they should take their most time and make every effort to learn. So, they should only take part in some community serives which needs less time they spend and is good for their development.

In a nutshell, taking part in community services have a positive impact on students development and help them success in the future. But parents and teachers should shoulder their responsibility to make sure the community services students take part in is not harmful for student`s health and developemt.
